iothub/device/src/Exceptions/IotHubClientException.cs

```diff
@@ -35,7 +35,7 @@ public class IotHubClientException : Exception
         // the hub service, so it is not always possible to obtain the HTTP status code and a specific error code.
         // With a best effort match, we map the error codes here to what the hub service exception has, and then match 
         // an appropriate HTTP status code for each of them via the dictionary below.
-        private static readonly Dictionary<IotHubClientErrorCode, HttpStatusCode> httpStatusCodes = new()
+        private static readonly Dictionary<IotHubClientErrorCode, HttpStatusCode> s_httpStatusCodes = new()
```

Comment:
Looks like we can get rid of HttpStatus code, per our v2 meeting. Sorry for asking you to do this work.

Reply:
No worries! We can always add it back in the future if necessary, better than keeping it now but deleting it further as a breaking change.